service/resource/configmapv2/delete_test.go

```diff
@@ -21,21 +21,21 @@ func Test_Service_newDeleteChange(t *testing.T) {
 	}{
 		// Test 0.
 		{
-			Obj: &v1alpha1.Ingress{
-				Spec: v1alpha1.IngressSpec{
-					GuestCluster: v1alpha1.IngressSpecGuestCluster{
+			Obj: &v1alpha1.IngressConfig{
```

Comment:
I did the type fixing here as well to get it working. In the other operators I will split this again. <3